methcomp/blandaltman.py

```diff
@@ -115,76 +123,44 @@ def plot(self, ax: matplotlib.axes.Axes):
         ax.spines['top'].set_visible(False)
 
         # set X and Y limits
-        if self.xlim is not None:
-            ax.set_xlim(self.xlim[0], self.xlim[1])
-        if self.ylim is not None:
-            ax.set_ylim(self.ylim[0], self.ylim[1])
+        if xlim is not None:
+            ax.set_xlim(xlim[0], xlim[1])
+        if ylim is not None:
+            ax.set_ylim(ylim[0], ylim[1])
 
         # graph labels
-        ax.set_ylabel(self.y_title)
-        ax.set_xlabel(self.x_title)
-        if self.graph_title is not None:
-            ax.set_title(self.graph_title)
+        ax.set_ylabel(y_title)
+        ax.set_xlabel(x_title)
+        if graph_title is not None:
+            ax.set_title(graph_title)
 
 
 def blandaltman(method1, method2,
```

Comment:
Actually, I think returning ax is correct, now you don't break backwards compatibility. A sneaky way to have both is to have an argument `return_object=False`, which could be used like this `return (ax, ba)  if return_object else ax`

Reply:
in the bladnaltman method or BlandAltman.plot() method?

Reply:
in `blandaltman` i think returning BlandAltman from plot would be a little crazy ;D